inc/customizer/colors.php

```diff
@@ -377,7 +386,7 @@ public function __construct() {
 					'default' => '#0b3954',
 					'section' => 'colors-menu',
 					'css'     => array(
-						'.main-navigation-container, .main-navigation.open, .main-navigation ul ul, .main-navigation .sub-menu' => array(
+						'.main-navigation-container, .main-navigation.open, .main-navigation ul ul, .main-navigation .sub-menu, .woocommerce .product span.onsale' => array(
```

Comment:
@EvanHerman I think the `.onsale` badge should use the Button Background color for its background, and Button Text Color for its text.